pom.xml

```diff
@@ -143,6 +143,30 @@
   </profiles>
 
   <build>
+
+    <!-- src distribution -->
```

Comment:
FYI: if we are not doing a concurrent release of all components, we might need to create separate source packages on a per-component basis. I'd keep as-is for now, and cross that bridge later.

No action needed.